File: LeetCode/0026_RemoveDuplicatesFromSortedArray/Solution.java
public class Solution {
    public int removeDuplicates(int[] nums) {
        int curr = 1;
        
        if (nums.length == 0 || nums == null)
            return 0;
        
        if (nums.length == 1)
            return 1;
        
        
        for (int i = 1; i<nums.length; i++){
            if (nums[i] != nums[i-1])
                nums[curr++] = nums[i];
        }
        
        return curr;
    }
}

File: LeetCode/0026_RemoveDuplicatesFromSortedArray/Solution.py
class Solution:
    # @param a list of integers
    # @return an integer
    def removeDuplicates(self, A):
        """
        :type nums: List[int]
        :rtype: int
        """
        if not A:
            return 0

        i, idx = 0, 1

        while idx < len(A):
            if A[i] != A[idx]:
                i += 1
                A[i] = A[idx]

            idx += 1

        return i + 1
